pkgs/development/compilers/zig/setup-hook.sh

```diff
@@ -0,0 +1,90 @@
+# shellcheck shell=bash disable=SC2154,SC2086
+
+# This readonly zigDefaultBuildFlagsArray below is meant to avoid CPU feature
+# impurity in Nixpkgs. However, this flagset is "unstable": it is specifically
+# meant to be controlled by the upstream developer, and she is not obliged to
```

Comment:
Nit and tangential: isn't "they" more inclusive?

Genuinely curious here, I'm not trying to start a flamewar.

Reply:
'They' is definitely better but not everyone is an native English speaker, so it's understandable.